transit_odp/pipelines/pipelines/dataset_etl/utils/transform.py

```diff
@@ -47,10 +62,19 @@ def transform_service_pattern_stops(
     service_pattern_stops = service_pattern_stops.where(
         service_pattern_stops.notnull(), None
     )
-
     return service_pattern_stops
 
 
+def create_flexible_stop_sequence(df: pd.DataFrame):
```

Comment:
Seems like this method is not used anymore

Reply:
done